src/api/controllers/articleController.js

```diff
@@ -56,6 +55,24 @@ export default class ArticleController {
     }
   }
 
+  /**
+   *
+   *
+   * @static
+   * @param {Object} req - request object
+   * @param {Object} res - respond object
+   * @returns {Object} response body
+   * @memberof ArticleController
+   */
+  static async delete(req, res) {
+    const { slug } = req.params;
+    await Article.destroy({ where: { slug } });
+    res.status(status.OK).json({
+      status: res.statusCode,
```

Comment:
where are you reading the statusCode from I saw that you changed it in the imported variable to status

Reply:
so I don't think the status response will be available